app/Http/Controllers/GroupController.php

```diff
@@ -23,6 +25,34 @@ public function __construct()
         $this->beforeFilter('auth', array('on' => array('post', 'put', 'delete')));
     }
 
+    /**
+     * Store newly created Group.
+     *
+     * @return Response
+     */
+    public function store(StoreGroupRequest $request)
```

Comment:
This should probably be named `create` to match with other routes?  Unless I've missed something here.